src/network/AuthenticationService.cpp

```
@@ -6,66 +6,163 @@
                                                QObject *parent)
     : QObject(parent)
+    , m_providerAdapters{providerAdapter}
     , m_transport(transport)
     , m_providerAdapter(providerAdapter)
     , m_requestFactory(providerAdapter ? providerAdapter->requestFactory() : nullptr)
     , m_providerAuthenticator(providerAdapter ? providerAdapter->authenticator() : nullptr)
     , m_secretStore(secretStore)
+{
+    configureTransport();
+}
+
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_UNUSED is a macro then please configure it.

(unknownMacro)